File: Kevin van den Bekerom/DSLQL/src/main/java/org/uva/sea/ql/ast/expr/GEq.java
package org.uva.sea.ql.ast.expr;

public class GEq extends Expr{
	Expr lhs, rhs;
	
	public GEq(Expr lhs, Expr rhs) {
		this.lhs = lhs;
		this.rhs = rhs;
	}
	
	@Override
	public Boolean eval() {
		return (Integer) lhs.eval() >= (Integer) rhs.eval();
	}
}

File: Edward Poot/src/ql/ast/expr/binaryoperatorexpr/geq.go
package binaryoperatorexpr

import (
	"ql/ast/expr"
)

type GEq struct {
	Lhs, Rhs expr.Expr
}

func (g GEq) Eval(s interface{}) interface{} {
	return g.Lhs.Eval(s).(int) >= g.Rhs.Eval(s).(int)
}
